### PR TITLE
Replace bytes_serialized with bogosize

### PR DESCRIPTION
**Pull request by sipa**
Current result:

```
  "bogosize": 3846482678,
  "disk_size": 2272310093,
```

### CODE REVIEW

**Comment by fanquake** [comment]
Looks like you'll need to update the blockchain RPC test [here](https://github.com/bitcoin/bitcoin/blob/master/test/functional/blockchain.py#L52).

**Comment by sipa** [comment]
Fixed.

**Comment by MarcoFalke** [comment]
Breaking changes need to be mentioned in the release notes. Mind to do it
in this pull?

**Comment by sipa** [comment]
I've rebased this PR on top of #10396, as both touch the same code. A single combined release-notes.md entry was added.

**Comment by sipa** [comment]
Rebased, and updated the release notes a bit.

**Comment by sipa** [comment]
Rebased on top of #10195.

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -904,6 +907,7 @@ UniValue gettxoutsetinfo(const JSONRPCRequest& request)
             "  \"bestblock\": \"hex\",   (string) the best block hash hex\n"
             "  \"transactions\": n,      (numeric) The number of transactions\n"
             "  \"txouts\": n,            (numeric) The number of output transactions\n"
+            "  \"bogosize\": n,          (numeric) A meaningless metric for UTXO set size\n"
             "  \"hash_serialized\": \"hash\",   (string) The serialized hash\n"
```
Comment (MarcoFalke) [inline src/rpc/blockchain.cpp]: Could you suffix the name here with `2` while touching the file anyway?

Comment (sipa) [inline src/rpc/blockchain.cpp]: Done.

**Comment by laanwj** [comment]
tested ACK 8b22af3

**Pull request merged**